packages/format-library/src/italic/index.js

```diff
@@ -10,9 +10,8 @@ const name = 'core/italic';
 export const italic = {
 	name,
 	title: __( 'Italic' ),
-	match: {
-		tagName: 'em',
-	},
+	tagName: 'em',
+	className: null,
 	edit( { isActive, value, onChange, ToolbarButton, Shortcut } ) {
```

Comment:
We're receiving components as props? That's really weird, I guess it's out of scope of this PR, but I'm interested in revisiting this pattern.

Reply:
Yes, we should change it. Maybe even before releasing?